src/test/lambda/commands/deleteLambda.test.ts

```diff
@@ -76,29 +81,47 @@ describe('deleteLambda', async () => {
             async configuration => assert.fail()
         )
 
-        await deleteLambda(node, () => {})
+        await deleteLambda({
+             node,
+             lambdaClient,
+             outputChannel: new MockOutputChannel(),
+             onRefresh: () => refreshCount++,
+             onConfirm: async () => confirmationResponse
+        })
+
+        const expectedDeleteCount = confirmationResponse ? 1 : 0
```

Comment:
It's easier to read/maintain the tests if the tests themselves set the expectations. Please pass in `expected*Count` from the individual tests.

Reply:
No `assertLambdaDeleteWorks` is a function which encapsulates logic to verify that that deleting works when expected. Expected delete count, refresh count, and any applicable output verified are internal implementation detail of the helper function. Prefer not to have leaky abstraction.

Reply:
Took your point to heart. Ultimately it would be ideal if we could quickly see the input and output factors of the test cases at a glance without having to dig through the code. Re-factored to do so.